### DISCUSSION TITLE
在windows下无法启动

### DISCUSSION THREAD

**Discussion by micahjiang2008** [illacloud/illa-builder#359] [Q&A] [answered]
hello
我在windows环境下运行illa deploy --self无法正常启动。错误如下：
“illa deploy --self
🔨  Running a self-hosted installation...
✅  Downloaded in 66 seconds
❌  Create ILLA Builder error: Docker responded with status code 400: invalid mount config for type "bind": bind source path”

环境：
windows10
wsl2
ubuntu
docker for windows

在docker里手动启动illasoft/illa-builder会报告无法连结数据库。localhost:80也无法访问

非常抱歉我对docker不是很熟悉，是否可以指教一下，多谢。

**Comment by naj1n** [comment]
@micahchiang2022 please provides the version number of illa cli.

**Reply by naj1n** [reply]
u can use `illa --version` command to get the version of illa cli.

**Reply by micahjiang2008** [reply]
illa 1.2.0
tks

**Reply by naj1n** [reply]
The latest version of illa is v1.2.2; maybe u can update it and try again.

**Reply by fimars** [reply]
same env, v1.2.2 still broken.

**Reply by naj1n** [reply]
@fimars could u report a bug in [illa](https://github.com/illa-family/illa), we'll fix it as soon as possible.

**Reply by naj1n** [reply]
@micahchiang2022 @fimars Hi, we have fixed this error [illa #42](https://github.com/illa-family/illa/pull/42). Now, you can download the latest illa cli of version v1.2.3.

**Reply by micahjiang2008** [reply]
多谢

**Reply by micahjiang2008** [reply, marked as answer]
v1.2.4 顺利安装，多谢。
